internal/pkg/overlay/funcmap.go

```diff
@@ -122,3 +123,12 @@ func createIgnitionJson(node *node.NodeConf) string {
 	tmpYaml, _ := json.Marshal(&conf)
 	return string(tmpYaml)
 }
+
+func importSoftlink(lnk string) string {
+	path, err := filepath.EvalSymlinks(lnk)
+	if err != nil {
+		return "abort"
+	}
+	wwlog.Debug("importing softlink pointing to: %s", path)
+	return fmt.Sprintf("{{ /* softlink \"%s\" */ }}", path)
+}
```

Comment:
Could this be replaced with a value that reads the target of a symlink, which could then be passed to the other symlink function? That justifies both functions (by using them both in the example localtime use case) and is more flexible for other templates later.